From 318a02b1fa7b9098e8471c04b1cfbf988c8b872e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Thu, 15 May 2025 11:38:42 +0200
Subject: [PATCH 839/954] refactor(stm32mp2): update SPDX license
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

update SPDX license on device tree to match Linux ones.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 107f49a26aa437c7e7bd958727b26828bad7801f refactor(stm32mp2): update SPDX license]
Change-Id: I565ad602f72c0cd2edf86be86c0e2b5086c347f4
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/458769
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 fdts/stm32mp21xxal-pinctrl.dtsi | 2 +-
 fdts/stm32mp21xxam-pinctrl.dtsi | 2 +-
 fdts/stm32mp21xxan-pinctrl.dtsi | 2 +-
 fdts/stm32mp21xxao-pinctrl.dtsi | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/fdts/stm32mp21xxal-pinctrl.dtsi b/fdts/stm32mp21xxal-pinctrl.dtsi
index 062284f516..9a7b13db1a 100644
--- a/fdts/stm32mp21xxal-pinctrl.dtsi
+++ b/fdts/stm32mp21xxal-pinctrl.dtsi
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: (GPL-2.0-or-later OR BSD-3-Clause)
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 /*
  * Copyright (C) STMicroelectronics 2024 - All Rights Reserved
  * Author: Alexandre Torgue <alexandre.torgue@foss.st.com> for STMicroelectronics.
diff --git a/fdts/stm32mp21xxam-pinctrl.dtsi b/fdts/stm32mp21xxam-pinctrl.dtsi
index 062284f516..9a7b13db1a 100644
--- a/fdts/stm32mp21xxam-pinctrl.dtsi
+++ b/fdts/stm32mp21xxam-pinctrl.dtsi
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: (GPL-2.0-or-later OR BSD-3-Clause)
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 /*
  * Copyright (C) STMicroelectronics 2024 - All Rights Reserved
  * Author: Alexandre Torgue <alexandre.torgue@foss.st.com> for STMicroelectronics.
diff --git a/fdts/stm32mp21xxan-pinctrl.dtsi b/fdts/stm32mp21xxan-pinctrl.dtsi
index 062284f516..9a7b13db1a 100644
--- a/fdts/stm32mp21xxan-pinctrl.dtsi
+++ b/fdts/stm32mp21xxan-pinctrl.dtsi
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: (GPL-2.0-or-later OR BSD-3-Clause)
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 /*
  * Copyright (C) STMicroelectronics 2024 - All Rights Reserved
  * Author: Alexandre Torgue <alexandre.torgue@foss.st.com> for STMicroelectronics.
diff --git a/fdts/stm32mp21xxao-pinctrl.dtsi b/fdts/stm32mp21xxao-pinctrl.dtsi
index 9b287e7811..b02f93cecd 100644
--- a/fdts/stm32mp21xxao-pinctrl.dtsi
+++ b/fdts/stm32mp21xxao-pinctrl.dtsi
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: (GPL-2.0-or-later OR BSD-3-Clause)
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
 /*
  * Copyright (C) STMicroelectronics 2024 - All Rights Reserved
  * Author: Alexandre Torgue <alexandre.torgue@foss.st.com> for STMicroelectronics.
-- 
2.53.0

